stoservice/src/main/java/com/tutrit/stoservice/provider/OrderProvider.java

```diff
@@ -0,0 +1,36 @@
+package com.tutrit.stoservice.provider;
+
+import com.tutrit.stoservice.bean.Order;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class OrderProvider {
+    private OrderProvider(){
+        throw new AssertionError("Suppress default constructor for noninstantiability");
+    }
+
+    static Map<Object, Object> getOrders() {
+        Map<Object, Object> initialOrders = new HashMap<>();
+        for (int i = 0; i < 10; i++) {
+            initialOrders.put(getInstance(i).getId(), getInstance(i));
```

Comment:
Two different objects are created here.
You can change 'getInstance(i).getId()' to '("id" + i)' to avoid creating unnecessary objects.